docs/examples/pod-ig.yaml

```diff
@@ -30,8 +29,6 @@ spec:
           name: host
         - mountPath: /run
           name: run
-        - mountPath: /sys/kernel/debug
```

Comment:
Could you please update [this documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/ig.md#using-ig-in-a-container) too?